### ISSUE TITLE
I still see albums with 0 tracks

### ISSUE THREAD

**Issue by ego-lay-atman-bay** [VinylMusicPlayer/VinylMusicPlayer#1128]
**Describe the bug**

On my phone, I have a ton of audio files from modding a game, so whenever I use an audio player, I have to use a blacklist of whitelist function to only look for songs in specific folders. The issue here is, I enabled the whitelist to only use songs from the Music folder, but since it had already scanned my entire device when I first installed it, the existing "albums" that were just random audio files, were still there, only they had 0 tracks and the app crashed if I tap and hold on the album.

**Device info**

 - Device: Google Pixel 6a
 - Android Version: 14
 - ROM: stock
 - Vinyl Music Player Version: 1.11.0

**Comment by GRbit** [comment]
Have you tried to run "Rescan library" after enabling whitelist?

**Comment by ego-lay-atman-bay** [comment]
Yes, that's the first thing I tried.

**Comment by ego-lay-atman-bay** [comment]
Actually, I'm not sure what's happening. I cleared the app data (to start fresh), and went through the setup process, while denying storage access, and then I enabled the whitelist, then allowed storage access, and now, on the first scan, I see albums with 0 tracks. I don't think it's empty folders, since nothing changed when I deleted an empty folder (and rescanned).

**Comment by lhardt** [comment]
Was the album name "Unknown Album"? May be the same issue as #1109, which seems to be songs with an album name but no artist name.

**Comment by ego-lay-atman-bay** [comment]
Yes, it is "unknown album", and in fact, that issue describes what I'm experiencing.
